src/Object/Api/Mastodon/Notification.php

```diff
@@ -44,9 +44,9 @@ class Notification extends BaseDataTransferObject
 	protected $created_at;
 	/** @var bool */
 	protected $dismissed;
-	/** @var Account */
+	/** @var array */
 	protected $account;
-	/** @var Status|null */
+	/** @var array|null */
 	protected $status = null;
```

Comment:
I think we should leave the strong types in, but update `BaseDataTransferObject->toArray()` to recursively call `toArray` on object properties instead of flatly using `get_object_vars()`.

Reply:
I would recommend to move this change into a separate PR. While refactoring I try to avoid such code changes.

Reply:
Fair enough